introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.            # if max_jobs=max_parallel_jobs then Bayesian search turns to Random.
</pre><p>If there's a concise way to explain the practical effect this might have on this model that might be helpful and/or interesting to readers - not a big deal, though. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3623/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>The explanation is the following :</p><p> </p><p>For Random Search there’s no cost. It’s just better to use a higher MaxParallelTrainingJobs (up to the limits) for maximum speed.&nbsp;</p><p> </p><p>For Bayesian search, the cost is that the higher MaxParallelTrainingJobs, the more similar Bayesian becomes to Random Search. The reason is that Bayesian won’t have all the sequential information to make the best decision on the hyperparameter to pick next. This means it may need more training jobs overall to find the optimum with a high parallel factor</p><p> </p><p>A bit hard to summarise the above in a concise way but part of it is in a way already mentioned. </p><p> </p>